dashboard/app/controllers/application_controller.rb

```diff
@@ -55,7 +55,7 @@ def configure_web_console
   def reset_session_endpoint
     client_state.reset
     reset_session
-    render text: "OK"
+    render text: 'OK <script>localStorage.clear()</script>'
```

Comment:
This sounds like a different bug?  The change here should be no different than the existing functionality for UI tests.